app/models/miq_request_workflow.rb

```diff
@@ -192,7 +192,12 @@ def validate(values)
         if fld[:required] == true
           # If :required_method is defined let it determine if the field is value
           unless fld[:required_method].nil?
-            fld[:error] = send(fld[:required_method], f, values, dlg, fld, value)
+            Array.wrap(fld[:required_method]).each do |method|
```

Comment:
I was tempted to change that as well, but unfortunately the behavior is different for `nil`; specifically it falls back to a default required method for validation, if defined (or it complains loudly if not defined).

Only vmware uses this:

```
jsselman@dev:~/manageiq (perform_instance_name_validation u= origin/perform_instance_name_validation)$ ag default_require app/
app/models/miq_request_workflow.rb
206:            default_require_method = "default_require_#{f}".to_sym
207:            if self.respond_to?(default_require_method)
208:              fld[:error] = send(default_require_method, f, values, dlg, fld, value)

app/models/miq_provision_virt_workflow/dialog_field_validation.rb
3:  def default_require_sysprep_enabled(_field, _values, dlg, fld, value)
9:  def default_require_sysprep_custom_spec(_field, _values, dlg, fld, value)
jsselman@dev:~/manageiq (perform_instance_name_validation u= origin/perform_instance_name_validation)$ ag sysprep_enabled product/
product/dialogs/miq_dialogs/miq_provision_dialogs.yaml
309:        :sysprep_enabled:

product/dialogs/miq_dialogs/miq_provision_vmware_dialogs_clone_to_template.yaml
309:        :sysprep_enabled:

product/dialogs/miq_dialogs/miq_provision_vmware_dialogs_clone_to_vm.yaml
318:        :sysprep_enabled:

product/dialogs/miq_dialogs/miq_provision_vmware_dialogs_template.yaml
309:        :sysprep_enabled:
jsselman@dev:~/manageiq (perform_instance_name_validation u= origin/perform_instance_name_validation)$
```

Alternatively I could remove the default logic and make the vmware fields explicitly define their validation method...